docs/tiltfile_concepts.md

```diff
@@ -217,22 +218,35 @@ k8s_resource(
 
 #### Kubernetes Object Selectors
 
-Tilt specifies Kubernetes objects via _Kubernetes object selectors_.
-The most specific selector for any object is its _fullname_: a colon-separated
-string specifying `$NAME:$KIND:$NAMESPACE` (e.g.: `redis:deployment:default`).
+You can specify Kubernetes objects via a _Kubernetes object selector_. (NOTE: 
+this is a Tilt-specific syntax. We wish Kubernetes already had a standard for
+specifying objects, but they don't, so we made our own.)
+
+The best-qualified object selector for a given object is a colon-separated
+string of the form `$NAME:$KIND:$NAMESPACE` (e.g.: `redis:deployment:default`).
+We call this the object's _fullname_.
 
 Additionally, you may specify an object using any prefix of its fullname. For above example,
```

Comment:
AIUI, you can't specify an object using a non-unique prefix. I might rephrase the first sentence to something like "Kubernetes object selectors also have shorter forms."

Reply:
yep, and i like this verbiage, ty